### PR TITLE
Bug 1122089 - Test the most played playlist

### PR DESCRIPTION
**Pull request by hfiguiere**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[hfiguiere Hubert Figuière (hfiguiere)](https://github.com/hfiguiere) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=468b9afcb086)

**Comment by try-server-hook** [comment]
[hfiguiere Hubert Figuière (hfiguiere)](https://github.com/hfiguiere) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=5dc91d6f648b)

File: apps/music/test/marionette/playlist_test.js [hunk 1]
```diff
@@ -276,5 +276,86 @@ marionette('Music player playlist', function() {
...
+        songData.metadata.played = 5;
+        w.musicdb.updateMetadata(songData.name, songData.metadata);
+
+        // 'Yield to Thread'
+        songData = w.SubListView.dataSource[0];
+        songData.metadata.played = 1;
+        w.musicdb.updateMetadata(songData.name, songData.metadata);
+
+        // 'Abort'
+        // Play count is 0 for that song.
```
Comment (davidflanagan) [inline apps/music/test/marionette/playlist_test.js]: Nit: the code in this executeScript call might be clearer if you re-ordered it to go from highest to lowest playcount.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b6265f61caccbbec46d057014b173a01f9ccb337

**Pull request merged**